cache/in-memory/src/model/guild.rs

```diff
@@ -123,8 +124,8 @@ impl CachedGuild {
     }
 
     /// ISO 8601 timestamp of the user's join date.
```

Comment:
ISO 8601 is not relevant since users choose their own formatting
```suggestion
    /// Timestamp of the user's join date.
```